oak_ml_transparency/mnist/README.md

```diff
@@ -0,0 +1,109 @@
+# The MNIST example
+
+This folder contains scripts for building an MNIST model and evaluating it
+against adversarial examples. The build script is only included here to make the
+example self-contained. The eval script should work with any MNIST model in
+[Tensorflow SavedModel format](https://www.tensorflow.org/guide/saved_model).
+
+## Building the Docker image
+
+We use a Docker image, with all the required dependencies installed to build the
+model, and run the evaluation. Having such a Docker image is necessary for
+achieving transparency. The Dockerfile should include the script for running the
+evaluation. More specifically, with the following lines in the Dockerfile, we
+embed the evaluation script in the Docker image:
+
+```bash
+RUN mkdir /project
+WORKDIR /project
+ADD . /project
+```
+
+Ideally, and as is the case in this example, the Dockerfile and the evaluation
+script should belong to the same repository, so that a single Git commit hash
+can uniquely specify both the content of the Dockerfile, and the script.
+
+Build the Docker image using the following command:
+
+```shell
```

Comment:
Is there a reason why some blocks are marked as `bash` and some as `shell`? I think they are the same.

Reply:
I meant to use `console` instead of shell. Fixed.